addons/storyshots/src/index.js

```diff
@@ -31,11 +31,14 @@ export default function testStorySnapshots(options = {}) {
   if (isStorybook) {
     storybook = require.requireActual('@storybook/react');
     // eslint-disable-next-line
-    const loadBabelConfig = require('@storybook/react/dist/server/babel_config').default;
+    const loadBabelConfig = require('@storybook/react/dist/server/babel_config')
+      .default;
     const configDirPath = path.resolve(options.configPath || '.storybook');
     configPath = path.join(configDirPath, 'config.js');
 
     const babelConfig = loadBabelConfig(configDirPath);
+    // We set this in the default babel config, but it is webpack-only
+    delete babelConfig.cacheDirectory;
```

Comment:
Potentially we should remove this config from this file: https://github.com/storybooks/storybook/blob/master/app/react/src/server/config/babel.js#L15= and instead add it when adding the config to webpack.